package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "dev": "webpack-dev-server",
+    "lint": "eslint . --ext js",
```

Comment:
If you run `npm run eslint` in console, it will run this command (`eslint . --ext js`). Here's the [documentation](https://eslint.org/docs/user-guide/command-line-interface) for the CLI options. It traverses every file in the project with _.js_ extension and spits out a detailed result of all syntax errors in the code
![image](https://user-images.githubusercontent.com/13035560/60785241-34e59f80-a120-11e9-8fea-ba2a0fe7f371.png)